src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -441,7 +440,7 @@ export function HeaderRow() {
 }
 
 /* Loading State: row component with loading bubbles */
-export function LoadingRow() {
+export function LoadingRow(props: { first?: boolean; last?: boolean }) {
```

Comment:
can you use CSS rules for this?

Reply:
i gave it a few tries and couldn't figure something out